corehq/apps/builds/models.py

```diff
@@ -166,7 +169,7 @@ def j2me_enabled_build_versions(cls):
 
 
 class BuildSpec(DocumentSchema):
-    version = StringProperty()
+    version = SemanticVersionProperty(required=False)
```

Comment:
This seems OK but why make this not required?

Reply:
When you add a new application object, the version is blank so this is to avoid failure there.